lib/service/user.go

```diff
@@ -279,3 +274,24 @@ func (svc *LndhubService) GetVolumeOverPeriod(ctx context.Context, userId int64,
 	}
 	return result, nil
 }
+
+func (svc *LndhubService) GetLimitsFromContext(c echo.Context) (limits *lnd.Limits) {
```

Comment:
I would create a function defined on `svc *LNDhubService` with the default limits, that are fetched from `svc.Config`. Then I would overwrite these values with the values from the context, if these are larger than 0 (since they will be set always they will never be nil)

Reply:
Changed to `GetLimits`